test/Error/rlexe.xml

```diff
@@ -178,4 +178,10 @@
       <compile-flags>-ExtendedErrorStackForTestHost -force:DeferParse</compile-flags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>Labels.js</files>
```

Comment:
I'm also curious if we have test coverage for these sorts of things elsewhere. If we do it probably dates back to ES5 and they may need to be updated or merged with this.

Reply:
There seem to be some label tests in core/test/Basics, these could all be condensed into one test file using the unit test framework, what do you think?

Reply:
That'd be awesome, those are probably ancient and need re-evaluation anyway.